src/cachers/index.js

```diff
@@ -2,18 +2,55 @@ if (process.env.NODE_ENV !== 'production') {
   require('dotenv').config();
 }
 
-const commands = [
-  'entities:organisations'
+const defu = require('defu');
+const utils = require('./utils');
+const nuxtConfig = require('../../nuxt.config');
+const runtimeConfig = defu(nuxtConfig.privateRuntimeConfig, nuxtConfig.publicRuntimeConfig);
+
+const cachers = [
+  'entities:organisations',
+  'entities:topics',
```

Comment:
this breaks the file on the current verion of the branch.
```
Error: Cannot find module './entities/topics'
```
Just commenting the line makes it work.

Reply:
Indeed. It's been removed.